Dockerfile

```diff
@@ -8,11 +8,6 @@ WORKDIR /srv/www
 COPY . .
 
 RUN apk add -U \
```

Comment:
@rupl remove this as well (keep `RUN`) or else it will throw errors.

should start with 
`RUN yarn install && \`